### DISCUSSION TITLE
No "Elektrische Wärmepumpenleistung" in HACS integration

### DISCUSSION THREAD

**Discussion by knell666** [LarsK1/hass_solvis_control#71] [General]
Hello Lars! Congratulations on your great work! I really like the integration for the Solvis heating system.
Unfortunately, the “Elektrische Wärmepumpenleistung” and “Leistung Wärmepumpe” are always 0. All other values are correct and comparable with the Solvis application. We have had our gas boiler switched off and a Mia 10KW heat pump connected instead. It was a conversion kit that was used. We also installed the SC3. Is there any way we can analyze and solve this problem together? The values are all visible in the standard Solvis application, so I'm sure they are there. 
Thanks Thomas

**Comment by LarsK1** [comment]
Hi Thomas,
which version is your Solvis. Please send a screenshot of Version SC2/3 and Version NBG.

P.S. we can also speak German if you like to...

**Comment by knell666** [comment]
Hi Lars,
das ist unsere Version von SC3 und NBG. Brauchst Du weitere Infos?
![Solvis Version](https://github.com/user-attachments/assets/1e3c4f60-46ad-4255-bd39-2548a0e69c86)

**Comment by LarsK1** [comment]
Hi Thomas,
bitte aktualisiere deine Heizung erst einmal auf die aktuelle Firmware. Siehe ab Seite 11: https://solvis-files.s3.eu-central-1.amazonaws.com/seiten/produkte/solviscontrol/Anleitung+Update+SC3+final.pdf

**Reply by knell666** [reply]
Hi Lars! 
Danke! Das war die Lösung. 
Jetzt werden alle Werte angezeigt.
Weiter so!
